pkgs/applications/audio/deadbeef/default.nix

```diff
@@ -108,7 +108,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Ultimate Music Player for GNU/Linux";
-    homepage = http://deadbeef.sourceforge.net/;
+    homepage = "http://deadbeef.sourceforge.net/";
```

Comment:
Unnecessary?

Reply:
we are slowly chipping away at implementing RFC 45 https://github.com/NixOS/rfcs/pull/45

Reply:
Interesting. I'm sure I've been told to remove the quotes before.

Reply:
thankfully now we have the RFC to get on the same page :)